doc/source/api.rst

```diff
@@ -1739,6 +1739,7 @@ Time/Date Components
    DatetimeIndex.nanosecond
    DatetimeIndex.date
    DatetimeIndex.time
```

Comment:
can add in the api section for Timestamp though

Reply:
The timetz method was already there for Timestamp, and I added the property for DatetimeIndex